sysbuild/Kconfig.suit

```diff
@@ -0,0 +1,95 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "SUIT"
+
+menuconfig SUIT_ENVELOPE
+	bool "Create SUIT envelope"
+	help
+	  Enable DFU SUIT envelope creation
+	default y if $(BOARD) = "nrf54h20dk_nrf54h20_cpuapp"
```

Comment:
move `default` up, help always goes last, and this doesn't work, you need to reference the BOARD_* symbol, or SOC_SERIES_* symbol (which would be better)